gruntfile.js

```diff
@@ -17,6 +17,9 @@ module.exports = function( grunt ) {
 
 		jshint: {
 			options: {
+				globals: {
+					'CKEDITOR': false
```

Comment:
Assigning `false` makes a perfect sense.

But as for using quotes for keys, I would second to @Reinmar.

I see that you'd like to give a special treatment to `dict` objects, but lets keep it simple. Object is an object, and the same rules apply everywhere.

Keeping consistency makes searching properties initializations much easier - you just make ctrl + f on whole directory with `<propName>:` and you hit all the occurences.